examples/units_and_coordinates/radec_to_hpc_map.py

```diff
@@ -6,9 +6,10 @@
 How to create a `~sunpy.map.Map` in Helioprojective Coordinate Frame from radio observations
 in GCRS (RA-DEC).
 
-In this example a LOFAR FITS file (made from CASA) is read in, the WCS
-header information is then used to make a new header with the information in
-Helioprojective, and a `~sunpy.map.Map` is made.
+In this example a LOFAR FITS file (created LOFAR's Default Pre-Processing Pipeline (DPPP) and
+WSClean Imager,  see `here <https://support.astron.nl/LOFARImagingCookbook/dppp.html>`_) is read in,
```

Comment:
```suggestion
WSClean Imager(`see here <https://support.astron.nl/LOFARImagingCookbook/dppp.html>`__) is read in,
```
?

Reply:
yeah maybe this link doesn't need to be here? @eoincarley who created the FITS file for this example suggests we include it.

Reply:
I am happy with the link.